src/utils/apis/JwtAPI.js

```diff
@@ -0,0 +1,18 @@
+import {recruitmentServiceBasicAPILink} from "./APILinks";
+import {loginAPI} from "./LoginAPI";
+import {jwtUtils} from "../jwt/jwtUtils";
+
+export const jwtAPI = {
+    refreshToken: (userId, refreshToken) =>
+        fetch(
+        recruitmentServiceBasicAPILink + jwtUtils.refreshApiPath(userId),
+        {
+                method: "POST",
```

Comment:
Rozważyłbym plik z funkcjami do nagłówków który by przyjmował tylko parametr jwtToken i jak go nie ma to by go nie dodawał do nagłówka, ale to nie koniecznie do tego taska